src/ui/components/VPNAlert.qml

```diff
@@ -317,27 +333,26 @@ Rectangle {
     function show() {
         if (!isLayout) {
             height = style.alertHeight;
-            width = Math.min(window.width - Theme.windowMargin, Theme.maxHorizontalContentWidth);
-            y = fullscreenRequired()? iosSafeAreaTopMargin.height + Theme.windowMargin : Theme.windowMargin;
+            width = window.width - Theme.windowMargin * 2;
```

Comment:
This is the only significant change for adjusting the width of the alert banner. The other changes are just minor formatting adjustments.